odoo/api.py

```diff
@@ -425,7 +425,7 @@ def model_create_multi(method):
 
 def _call_kw_model(method, self, args, kwargs):
     context, args, kwargs = split_context(method, args, kwargs)
-    recs = self.with_context(context or {})
+    recs = self.with_context(context or {}, call_kw=True)
```

Comment:
Don't understand this part/solution, and I strongly disprove it:

- The fact that the context is transmitted between environment. Then the distinction between "outside/inside" server environment is blurry.
- Having a context key modifying the behavior of a security check is "weird".
- Is it still needed ? Only `exportable=False` isn't enough? Context key should be only the last resort!

Reply:
My idea was to narrow number of cases when the patch blocks read access.

Without context it still fix the issue, but I'm not sure whether it introduces new problems.
And if it does, I don't know what could be alternative solution.

![image](https://user-images.githubusercontent.com/186131/224972965-6c950695-42f7-4c81-b84b-d289955367b8.png)